requirements.txt

```diff
@@ -1,4 +1,7 @@
-django-cms>=3.7,<3.8
+wheel
+
+#django-cms>=3.7,<3.8
+-e git+git://github.com/bcvandendool/django-cms@release/3.7.x#egg=django-cms
```

Comment:
@bcvandendool can you explain what this line does?

Reply:
it gets the modified django-cms we need from git and installs that instead of the default one from the pip repos

Reply:
what is modified exactly?

Reply:
it fixes the migrations to support multiple databases